### PR TITLE
Containerization

### PR DESCRIPTION
**Pull request by fredrbus**
## Related Issue
Closes [Containerization of project](https://trello.com/c/8gAp2jCH/7-containerization-of-project)

## Proposed changes
- Both frontend and backend can be run as Docker-containers

## Additional info
- any additional information or context 

## How has this been tested?
- [ ] All group members have tried and succeeded in running the containerized application

## Checklist
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by fredrbus** [comment]
Tests were expected to fail, as the ESLint/Prettier bugfix is not implemented in this branch

File: README.md [hunk 1]
```diff
@@ -8,12 +8,47 @@ A tool for visualizing ontologies related to UN's sustainable development goals
...
+
+2. Place your `graphdb.license` file in `backend/database/conf`
+
+3. Copy the `backend/.env.example`-file, and rename the copy `.env`
+
+4. In `backend/`, run `docker-compose -f docker-compose-backend.yml up`
+
+5. When the Docker-cluster is running, go to http://localhost:7200. On the left side of the screen, go to "Setup" and then "Users and Access". Click "Create new user" and make a user with credentials matching the `GRAPHDB_USERNAME` and `GRAPHDB_PASSWORD` fields in `backend/.env`. Make sure the user has Read-access to the TK_SDG-repository.
+
+6. Stop the docker-compose cluster, and repeat step 2.
```
Comment (Jorgro) [inline README.md:40]:
```suggestion
6. Stop the docker-compose cluster, and repeat step 4.
```

**Review by Jorgro — APPROVED** [review]
While I think this is good as an initial working containerization there is one thing I am lacking:

Use volumes instead of just copying the files into the container! Right now when we edit files on our own computer this won't reflect in the docker containers so to ease the development process a lot we should instead mount the files from our computer into the docker container. See here for how you can do this: https://stackoverflow.com/questions/60977357/how-do-i-mount-my-local-react-directory-into-my-react-docker-container Also if it isn't enabled yet we should have automatic hot-reloading. 

I made an issue on this here: https://trello.com/c/1gWYT4xl/22-use-volumes-in-docker

File: README.md (hunk 1 above)
Comment (Jorgro) [reply README.md:40]: I made an issue on this: https://trello.com/c/1gWYT4xl/22-use-volumes-in-docker

Comment (Jorgro) [reply README.md:40]: Clarification: Should create a **separate** docker compose for development as this is intended for deployment only. My mistake!

**Pull request merged**